bamboo/npm-tag.js

```diff
@@ -3,7 +3,11 @@
 'use strict';
 
 const fs = require('fs-extra');
-const getLatestVersion = require('latest-version');
+async function importGetLatestVersion() {
```

Comment:
this is to allay the publish step, which yells about bringing in latest version with require
the latest-version module does not support being brought in with require, and so this little band-aid